tenets/codelingo/code-review-comments/context-in-struct/codelingo.yaml

```diff
@@ -0,0 +1,36 @@
+tenets:
+  - name: context-in-struct
+    flows:
+      codelingo/review:
+        comment: Don't add a Context member to a struct type; instead add a ctx parameter to each method on that type that needs to pass it along.
```

Comment:
The comment should link to CRC.

Reply:
We need to justify our standard with an external reference so we don't get pulled into arguments.